samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,95 @@
-     If the application core has written an update instruction, copy the specified data range to the application partition on the network core.
-     Once the copy is done, compare the SHA of the data in the application partition against the SHA specified in the shared SRAM.
-     Communicate the result of the comparison to the application core through the shared SRAM.
-
-#. Lock the flash of the application.
-     Lock the flash that contains the application.
+     a. If the application core has written an update instruction, the bootloader copies the specified data range to the application partition on the network core.
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
+     #. It then communicates the result of the comparison to the application core through the shared SRAM.
+#. It then locks the flash memory areas containing the application.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
-
-#. Boot the application on the network core.
-     After possibly performing a firmware update, and enabling flash protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+#. It boots the application on the network core.
+     After possibly performing a firmware update, and enabling flash memory protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
 
 Requirements
 ************
 
-  * |nRF5340DK|
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpunet
 
 .. _net_bootloader_build_and_run:
 
 Building and running
 ********************
 
-The source code of the sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
+The recommended way of building the sample is to include it as a child image in a :ref:`multi-image <ug_multi_image>` build where there is an application that runs on the network core, rather than building it stand-alone.
+There are two ways to include it in a multi-image build:
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+* If you don't need the capability to update the sample, enable the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the network core.
```

Comment:
This point can be removed IMO.

This sample gives no value if MCUboot is not enabled on the app core. This because this sample is solely driven from the configuration written to shared SRAM by MCUboot. If MCUboot is not in place, this sample does nothing other than jumping to the application.

Reply:
fixed